src/main/webapp/app/entities/event-crew/event-crew.tsx

```diff
@@ -1,102 +1,101 @@
 import React from 'react';
 import { connect } from 'react-redux';
 import { Link, RouteComponentProps } from 'react-router-dom';
-import { Button, Col, Row, Table } from 'reactstrap';
+import { Container, Button, Col, Row, Table } from 'reactstrap';
 // tslint:disable-next-line:no-unused-variable
-import { Translate, ICrudGetAllAction } from 'react-jhipster';
+import { Translate, getSortState, ICrudGetAllAction, IPaginationBaseState } from 'react-jhipster';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 
+import { ITEMS_PER_PAGE } from 'app/shared/util/pagination.constants';
 import { IRootState } from 'app/shared/reducers';
-import { getEntities } from './event-crew.reducer';
-import { IEventCrew } from 'app/shared/model/event-crew.model';
+import { getEntities, getEventCrewByEventId } from './event-crew.reducer';
+import { IEventCrew, EventCrewRole } from 'app/shared/model/event-crew.model';
 // tslint:disable-next-line:no-unused-variable
 import { APP_DATE_FORMAT, APP_LOCAL_DATE_FORMAT } from 'app/config/constants';
+import { CustomTab } from 'app/shared/components/customTab/custom-tab';
+import EventModal from 'app/shared/components/eventModal/event-modal';
+import { eventTabList } from 'app/shared/util/tab.constants';
+import { EventTable } from 'app/shared/components/eventTable/EventTable';
 
-export interface IEventCrewProps extends StateProps, DispatchProps, RouteComponentProps<{ url: string }> {}
+export interface IEventCrewProps extends StateProps, DispatchProps, RouteComponentProps<{ url: string; eventId: string }> {}
 
 export class EventCrew extends React.Component<IEventCrewProps> {
+  state = {
+    ...getSortState(this.props.location, ITEMS_PER_PAGE),
+    modalIsOpen: false,
+    eventCrewId: null,
+    eventId: this.props.match.params.eventId
+  };
+
+  openModal = eventCrewId => {
+    this.setState({ modalIsOpen: true, eventCrewId });
+  };
+
+  closeModal = () => {
+    this.setState({ modalIsOpen: false, eventCrewId: null });
+  };
+
   componentDidMount() {
-    this.props.getEntities();
+    const { eventId } = this.props.match.params;
+    this.props.getEventCrewByEventId(Number.parseInt(eventId, 10));
   }
 
+  //   sort = prop => () => {
+  //     this.setState(
+  //       {
+  //         order: this.state.order === 'asc' ? 'desc' : 'asc',
+  //         sort: prop
+  //       },
+  //       () => this.sortEntities()
+  //     );
+  //   };
+
+  //   sortEntities() {
+  //     this.getEntities();
+  //     this.props.history.push(`${this.props.location.pathname}?page=${this.state.activePage}&sort=${this.state.sort},${this.state.order}`);
+  //   }
```

Comment:
Remove if not in use

Reply:
Resolved in dcaf9e6e87fbe5d28af246ab603fa71b1b0dd344